### DISCUSSION TITLE
Apache-airflow v2.5.1 version compatibility with Flask-AppBuilder version 4.3.0

### DISCUSSION THREAD

**Discussion by sainman** [apache/airflow#30721] [Q&A]
Hi

We are using Apache-airflow v2.5.1 which is using with Flask-AppBuilder version 4.1.4 and working well.

A few days back Flask-AppBuilder version 4.1.4 is marked as a vulnerable item with high severity by Dependabot,  and we have to upgrade it to version 4.3.0.

Now the problem is - Airflow v2.5.1 is not supporting Flask-AppBuilder version 4.3.0 ( any >4.1.4 )

Could anyone please suggest what could be a possible solution or timeline for Airflow will support Flask-AppBuilder version 4.3.0?

Thanks
![airflow](https://user-images.githubusercontent.com/74210150/232888136-e6720a6e-abb6-4386-872c-aa7593e88d58.JPG)

**Comment by potiuk** [comment]
Please try to avoid spamming multiple channels with the same question - especially if it has been answered already https://github.com/apache/airflow/issues/22397#issuecomment-1513696796 - copy pasting the same message in multiple places without being considerate for other people without even waiting few minutes for an answer (and continue doing so after receing it is one thing that distinguishes being persistent from those who spam people.
